### ISSUE TITLE
Error Installing VAST "cannot remove prior installation of package x"

### ISSUE THREAD

**Issue by aczich** [James-Thorson-NOAA/VAST#282]
Upon trying to install VAST I've run into the following error when installing from both github and local zip files. This has been tested on 3 separate computers, all failing with the same error message. 

install_github("james-thorson/VAST", INSTALL_opts="--no-staged-install")

Error: Failed to install 'VAST' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘cachem’

This error isn't specific to the package 'cachem', on other computers it has been the package 'fansi', 'processx' and 'glue'

**Comment by James-Thorson-NOAA** [comment]
Hmm.  you could try

```R
install_github("james-thorson/VAST", INSTALL_opts="--no-staged-install", dep=FALSE)
```

and then separately installing those dependencies as needed, e.g., whenever the terminal references specific packages that are missing?

**Comment by aczich** [comment]
Just tried and I've got a new error message. I also installed cachem separately with the same result below. 
Error: Failed to install 'VAST' from GitHub:
  (converted from warning) installation of package ‘C:/Users/User/AppData/Local/Temp/RtmpItr4VH/file2050385150c2/VAST_3.6.1.tar.gz’ had non-zero exit status

**Comment by aczich** [comment]
Update: I ran into several issues along the way. The main one was that R creates 00LOCK folders for the failing packages. First I manually deleted those folder and then made sure those packages were installed, then installed the package and selecting update none (enter 3 in the console) . Then I ran into issues due to error messages related to TMB, I would suggest running 

install.packages('TMB',type='source')

however, this did not work for me and so I installed the package using

install_github('cran/TM', , INSTALL_opts="--no-staged-install") and once again chose to update none.
Then, tried installing VAST once more and selecting to update none and it successfully installed. 

This was also all done in vanilla R.
